src/burp/restore.epp

```diff
@@ -8906,6 +8928,12 @@ void get_source_blob(BurpGlobals* tdgbl, ISC_QUAD& blob_id, bool glb_trans)
 		--length; // -- or ++ ??? p is decremented, will have to test.
 		const USHORT seg_len = p - buffer;
 
+		if (postProcess)
+		{
+			postProcess(static_buffer);
```

Comment:
Looks like this is wrong place to have this, as it's inside a loop related to segments.

Reply:
On practice we never have specific attributes with >1 segment. Well, let's be generic & analyze all blob as a whole. Will do.